<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="10,0,0,0,0">
   <item row="5" column="0" colspan="5">
    <widget class="QLabel" name="labelDebug"/>
   </item>
   <item row="7" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Help|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Micro break</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QTimeEdit" name="timeEditRestBreak">
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::HourSection</enum>
     </property>
     <property name="displayFormat">
      <string>HH:mm:ss</string>
     </property>
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>10</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QTimeEdit" name="timeEditRestBreakEvery">
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::HourSection</enum>
     </property>
     <property name="displayFormat">
      <string>HH:mm:ss</string>
     </property>
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>45</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>every</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>every</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QTimeEdit" name="timeEditMicroBreak">
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::HourSection</enum>
     </property>
     <property name="displayFormat">
      <string>HH:mm:ss</string>
     </property>
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>1</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Rest break</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QTimeEdit" name="timeEditMicroBreakEvery">
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::HourSection</enum>
     </property>
     <property name="displayFormat">
      <string>HH:mm:ss</string>
     </property>
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>10</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,10,0,0">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Alert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxAlert"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Volume</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderVolume">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="checkBoxStartOnLogin">
     <property name="text">
      <string>Start on login</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>timeEditMicroBreak</tabstop>
  <tabstop>timeEditMicroBreakEvery</tabstop>
  <tabstop>timeEditRestBreak</tabstop>
  <tabstop>timeEditRestBreakEvery</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
